_data/toc.yaml

```diff
@@ -1428,71 +1280,269 @@ manuals:
+      title: SLES
     - path: /engine/install/ubuntu/
-      title: Install on Ubuntu
+      title: Ubuntu
     - path: /engine/install/binaries/
-      title: Install binaries
+      title: Binaries
     - path: /engine/install/linux-postinstall/
       title: Post-installation steps
     - path: /engine/install/troubleshoot/
       title: Troubleshoot installation
+  - sectiontitle: Storage
+    section:
+    - path: /storage/
+      title: Overview
+    - path: /storage/volumes/
+      title: Volumes
+    - path: /storage/bind-mounts/
+      title: Bind mounts
+    - path: /storage/tmpfs/
+      title: tmpfs mounts
+    - path: /storage/troubleshooting_volume_errors/
+      title: Troubleshoot
+    - sectiontitle: Storage drivers
+      section:
+      - path: /storage/storagedriver/
+        title: Overview
+      - path: /storage/storagedriver/select-storage-driver/
+        title: Select a storage driver
+      - path: /storage/storagedriver/aufs-driver/
+        title: Use the AUFS storage driver
+      - path: /storage/storagedriver/btrfs-driver/
+        title: Use the Btrfs storage driver
+      - path: /storage/storagedriver/device-mapper-driver/
+        title: Use the Device mapper storage driver
+      - path: /storage/storagedriver/overlayfs-driver/
+        title: Use the OverlayFS storage driver
+      - path: /storage/storagedriver/zfs-driver/
+        title: Use the ZFS storage driver
+      - path: /storage/storagedriver/vfs-driver/
+        title: Use the VFS storage driver
+  - sectiontitle: Networking
+    section:
+    - path: /network/
+      title: Overview
+    - path: /config/containers/container-networking/
+      title: Container networking
+    - path: /network/proxy/
+      title: Configure Docker to use a proxy server
+    - path: /network/bridge/
+      title: Bridge networks
+    - path: /network/overlay/
+      title: Overlay networks
+    - path: /network/host/
+      title: Host networking
+    - path: /network/ipvlan/
+      title: IPvlan networks
+    - path: /network/macvlan/
+      title: Macvlan networks
+    - path: /network/none/
+      title: Disable networking for a container
+    - sectiontitle: Networking tutorials
+      section:
+      - path: /network/network-tutorial-standalone/
+        title: Bridge network tutorial
+      - path: /network/network-tutorial-host/
+        title: Host networking tutorial
+      - path: /network/network-tutorial-overlay/
+        title: Overlay networking tutorial
+      - path: /network/network-tutorial-macvlan/
+        title: Macvlan network tutorial
+    - sectiontitle: Legacy networking content
+      section:
+      - path: /network/links/
+        title: (Legacy) Container links
+  - sectiontitle: Working with Docker Engine
+    section:
+    - path: /config/daemon/start/
+      title: Start the daemon
+    - path: /config/pruning/
+      title: Prune unused objects
+    - path: /config/formatting/
+      title: Format command and log output
```

Comment:
For a follow-up; this is a topic we should have somewhere central to "using the cli".